http/headers/Sec-Speculation-Tags.json

```
@@ -0,0 +1,43 @@
+{
+  "http": {
+    "headers": {
+      "Sec-Speculation-Tags": {
+        "__compat": {
+          "spec_url": "https://wicg.github.io/nav-speculation/speculation-rules.html#speculation-rules-header",
```

This `spec_url` refers to the `Speculation-Rules` header, not the `Sec-Speculation-Tags` header.

---

Agreed. And I _think_ you're correct with your updated spec the way it's defined now.

What's confusing me is that this header is applicable to both Speculation Rules Prefetch and Speculation Rules Prerender. Yet it only exists in https://wicg.github.io/nav-speculation/prefetch.html#sec-speculation-tags-header and not https://wicg.github.io/nav-speculation/prerendering.html.

Asking the spec owners what the deal is here or if it should be defined in the wider https://wicg.github.io/nav-speculation/speculation-rules.html spec that governs both.

---

Can probably merge it with your suggested change even if it needs spec updates, as it's correct for now, but let's see if I can quickly get a definitive answer here first.

Adding back your change for now:

```suggestion
          "spec_url": "https://wicg.github.io/nav-speculation/prefetch.html#sec-speculation-tags-header",
```

---

Got it. If the header ends up being defined in both specs, than we can set `spec_url` to an array pointing to both. But in general, we want the `spec_url` to be specific to the feature.

---

Yup agreed. And good spot for noticing it was wrong!

---

FYI, I got confirmation that IS the correct spec URL, even though it replies to both.

Basically the prerender spec references the prefetch spec for things like this when it says "11 [Prefetch](https://wicg.github.io/nav-speculation/prefetch.html#prefetch) given `referrerDoc` and `prefetchRecord`."

---

@tunetheweb Which one is the correct spec URL?

https://wicg.github.io/nav-speculation/speculation-rules.html#speculation-rules-header
or
https://wicg.github.io/nav-speculation/prefetch.html#sec-speculation-tags-header

---

The second one is the correct one as you suggested. And that's what was merged.